### DISCUSSION TITLE
Unable to download Aluminium Profile Surface Defect Dataset

### DISCUSSION THREAD

**Discussion by stepanje** [Charmve/Surface-Defect-Detection#6] [General]
Hi,

I am trying to download the Aluminium Profile Surface Defect Dataset, Tianchi requires registration to the competition to download the dataset, but the registration does not seem to work because the competition has already ended. Is it possible to download the dataset somewhere else? Thank you for doing this repository and for any tips about the dataset!

**Comment by Charmve** [comment]
Hi, stepanje

Could you download the dataset from this official url, https://tianchi.aliyun.com/competition/entrance/231682/information

![image](https://user-images.githubusercontent.com/29084184/119592659-7c0bd500-be0b-11eb-8a37-b643e6423093.png)

Register and log in.

@Charmve

**Reply by stepanje** [reply]
I tried registering to the AlibabaCloud as required, but when I logged in, I started getting this message

![dset](https://user-images.githubusercontent.com/43796129/119642653-3a832600-be1b-11eb-8a2f-2d3c642e829a.jpg)

and I could not find any way to register to this particular competition. I think this may be because the competition has already ended. Could that be the case or is there still any way to get to the dataset?

**Comment by kellyzpz** [comment]
weblink：https://pan.baidu.com/s/1jnSwJ2xRzdSplUtvTbuIuw

password：i10s

**Reply by NamLam-L** [reply]
Hi, I've just  downloaded the dataset. However, I can not extract the  guangdong_round2_train_20181011.zip and guangdong_round2_train_201810111.zip since missing the file guangdong_round2_train_20181011.z01 and guangdong_round2_train_201810111.z01. Can you provide me these two files. Thanks!!

**Comment by Hrithik2212** [comment]
@kellyzpz  Hey , I am from India and unable to open this particular link , not sure why. Is there any other way I can get this dataset
